composer.json

```diff
@@ -28,10 +28,10 @@
 	},
 	"require": {
 		"php": "~7.2.0 || ~7.3.0 || ~7.4.0 || ~8.0.0 || ~8.1.0",
-		"typo3/cms-core": "^10.4 || ^11.5.2",
-		"typo3/cms-extbase": "^10.4.6 || ^11.5.2",
-		"typo3/cms-fluid": "^10.4 || ^11.5.2",
-		"typo3/cms-frontend": "^10.4 || ^11.5.2"
+		"typo3/cms-core": "^10.4.11 || ^11.5.2",
```

Comment:
It should suffice to raise the version of core - the other extension can stay at their minimum requirements.

Reply:
Does that really make sense? You cannot use a different version of typo3/cms-core and typo3/cms-extbase anyway. And then why for example do we also require certain bug-fix versions for the other system extensions for TYPO3 v11?

Reply:
The V11 thing indeed is an inconsistency. I’d like to discuss our policy concerning how to handle system extension versions at our next call (of the best practices team).